tests/new_resolvers/sports_resolvers/raw_sports/raw_sports_teams_key/test_sport_lab_teams.py

```
@@ -47,7 +47,7 @@
     "world amateur handball championship": "بطولة العالم لكرة اليد للهواة",
     "world aquatics championships competitors": "منافسو بطولة العالم للرياضات المائية",
     "world archery championships": "بطولة العالم للنبالة",
-    "world athletics championships ": "بطولة العالم لألعاب القوى",
+    "world athletics championships": "بطولة العالم لألعاب القوى",
     "world athletics championships medalists": "فائزون بميداليات بطولة العالم لألعاب القوى",
     "world athletics championships": "بطولة العالم لألعاب القوى",
```

Ruff (0.14.13)

Dictionary key literal `"world athletics championships"` repeated

Remove repeated key literal `"world athletics championships"`

(F601)